### PR TITLE
feat : filters page

### PR DESCRIPTION
**Pull request by rohit8686**
Features : 

- Sort notes by oldest or newest
- filter notes based on labels
- search notes
- clear filters

### CODE REVIEW

**Review by surajmore98 — APPROVED** [review]
well structured code.
good use of context hook.

File: src/frontend/pages/Notes/Labels/Labels.jsx [hunk 1]
```diff
@@ -2,18 +2,20 @@ import React from "react";
 import { ToastContainer } from "react-toastify";
 import { useArchive } from "../../../contexts/archive-context";
 import { useEdit } from "../../../contexts/edit-context";
+import { useFilter } from "../../../contexts/filters-context";
 import { useNote } from "../../../contexts/note-context";
```
Comment (anandsr21harshit) [inline src/frontend/pages/Notes/Labels/Labels.jsx:6]: create an index file and use barrel import

File: src/frontend/pages/Notes/Search/search.css [hunk 2]
```diff
@@ -15,3 +15,6 @@
 .search-icon {
   color: var(--black);
```
Comment (anandsr21harshit) [inline src/frontend/pages/Notes/Search/search.css:16]: what's the point of using variable name if you are giving it as color name?

**Review by anandsr21harshit — CHANGES_REQUESTED** [review]
Create barrel import for good readability

File: src/frontend/pages/Notes/Labels/Labels.jsx (hunk 1 above)
Comment (rohit8686) [reply src/frontend/pages/Notes/Labels/Labels.jsx:6]: yeah will do it in refactor branch

**Review by anandsr21harshit — APPROVED** [review]

**Pull request merged**